legos/cryptocurrency.py

```diff
@@ -2,15 +2,28 @@
 import requests
 import logging
 import json
+import configparser
 
 logger = logging.getLogger(__name__)
 
 
 class Cryptocurrency(Lego):
+    def __init__(self, baseplate, lock):
+        super().__init__(baseplate, lock)
+        self.crypto_index = configparser.ConfigParser()
+        self.crypto_index.read('crypto_index.ini')
+        if 'crypto_index' not in self.crypto_index.sections():
+            self.crypto_index['crypto_index'] = {}
+            self.crypto_index['crypto_index']['fsyms'] = 'BTC'
```

Comment:
the Cryptowatch may have used the horribly cryptic `fsyms` and `tsyms`, but that doesn't mean we have to. It would probably benefit the user if we made these terms more clear and documented what they correspond to.

Reply:
resolved.
changed crypto_index to hodl
changed fsyms to convert_from
changed tsyms to convert_to